ts/parts/Interaction.ts

```diff
@@ -456,7 +456,7 @@ extend(Legend.prototype, {
 
         // Set the events on the item group, or in case of useHTML, the item
         // itself (#1249)
-        (useHTML ? legendItem : (item.legendGroup as any))
+        itemToSet
```

Comment:
Instead of changing the function parameters and the way they are called, I think it would be cleaner just to change it here. Something like this:

```js
const elements = useHTML ? [legendItem, item.legendSymbol] : [item.legendGroup];
elements.forEach(element => {
    element.on('mouseover' ......
```